src/Bridges/Constraint/ltgt_to_interval.jl

```diff
@@ -0,0 +1,73 @@
+# The code here is mostly copied from the flip_sign.jl code for FlipSignBridge and GreaterToLessBridge
+
+"""
+    AbstractToIntervalBridge{T, S1, F}
+
+Bridge a `F`-in-`S1` constraint into an `F`-in-`Interval` constraint where we have either:
+* `S1 = MOI.GreaterThan{T}`
+* `S1 = MOI.LessThan{T}`
+
+The `F`-in-`S1` constraint is stored in the `constraint`
+field by convention.
+"""
+abstract type AbstractToIntervalBridge{
+    T, S1<:MOI.AbstractSet,
```

Comment:
`T<:AbstractFloat`

Reply:
Add a comment that `typemin` and `typemax` either is not implemented (e.g. `BigInt`) or does not give infinite value (e.g. `Int`) for non-`AbstractFloat` types

Reply:
Why does it need to be a float? It fails the test in `test/instantiate.jl` for `Instantiate with Int`

Reply:
Never mind me. I remembered it's because of the Inf for int being undefined.
About the failed test, how can I tell MOI to skip this bridge if T is not a float? I would have expected that julia would automatically skip the bridge if it's passing an int and this bridge is implemented for float.